### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/) for ChainSafe Files/Chainsafe Files user interface.

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by RyRy79261** [comment]
So theres no changes here 😅

**Comment by FSM1** [comment]
> So theres no changes here 😅

I think that this is due to the fork merging all the existing PR's that were merged onto `dev`

**Comment by Tbaut** [comment]
yeah that bot isn't super clever :D
I tried to figure out why the other day, but failed at doing so, I'll take a look in the next days again, the settings of Weblate are far from easy to navigate!

File: packages/files-ui/src/locales/fr/messages.po [hunk 1]
```diff
@@ -69,7 +70,7 @@ msgid "Browser:"
 msgstr "Explorateur :"
 
 msgid "Bucket id"
-msgstr ""
+msgstr "Identifiant du seau"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po:76]:
```suggestion
msgstr "Identifiant du Bucket"
```

**Review by Tbaut — APPROVED** [review]

**Pull request merged**